views/address.tmpl

```diff
@@ -13,7 +15,8 @@
         <div class="row">
             <div class="col-md-8 col-sm-6">
                 <h4>Address</h4>
-                <div class="mono" data-target="address.addr" style="margin-bottom: 12px;">
+                <div class="mono" data-target="address.addr"
+                    id="{{$TxTime}}" style="margin-bottom: 12px;">
```

Comment:
I don't follow why this is tagged with `id="{{$TxTime}}"`.  Could you explain the idea?

Reply:
This variable helps in choosing the most appropriate `group by` interval that should have lesser payload size and also provide more meaningful chart information to the user by default (when user accesses the chart view for the first time).
Its also helps deactivate by default the `zoom` and `group by` buttons that are not applicable in relation to the current address's data set.
Used here https://github.com/decred/dcrdata/pull/633/files#diff-90ad06f5b024ce10091b98d9a3be7766R230

Reply:
I also wanted a way to tag that time with an existing data-target rather that create a new data-target for it.

Reply:
Ah, the `var val = parseInt(this.addrTarget.id)` was what I was missing.  Thanks!